app/src/main/java/com/x8bit/bitwarden/ui/platform/components/segment/BitwardenSegmentedButton.kt

```diff
@@ -23,17 +30,22 @@ import kotlinx.collections.immutable.ImmutableList
  *
  * @param options List of options to display.
  * @param modifier Modifier.
+ * @param windowInsets The insets to be applied to this composable.
  */
 @Composable
 fun BitwardenSegmentedButton(
     options: ImmutableList<SegmentedButtonState>,
     modifier: Modifier = Modifier,
+    windowInsets: WindowInsets = WindowInsets.displayCutout
```

Comment:
Do you feel there would be any value to having some of these as `Modifier.insetExtensionWithBetterNameThanThis()` ? I do see some of them are kind of specific to what component it is being applied to and where that shows up (FAB for example) but I feel like I saw this one a few times. If its just a couple that use this for the `windowInsetsPadding` then just ignore this :)

Reply:
This one is the same across most utilityBar and topAppBar items. I don't think I'd want a modifier, since we want it to be overridable but a predefined inset could be useful.